src/main/java/frc/utils/ToleranceCalculations.java

```diff
@@ -0,0 +1,17 @@
+package frc.utils;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.constants.MathConstants;
+
+public class ToleranceCalculations {
+
+	public static boolean isRotation2dInTolerance(Rotation2d tolerance, Rotation2d rotation, Rotation2d wantedRotation) {
+		return Math.abs(wantedRotation.getRadians() - rotation.getRadians()) <= tolerance.getRadians()
+			|| Math.abs(wantedRotation.getRadians() - rotation.getRadians()) >= MathConstants.FULL_CIRCLE.getRadians() - tolerance.getRadians();
+	}
+
+	public static boolean isDoubleInRange(double tolerance, double value, double maximumValue, double minimumValue) {
+		return (maximumValue + tolerance) >= value && (minimumValue - tolerance) <= value;
```

Comment:
```suggestion
		return (minimumValue - tolerance) <= value && value <= (maximumValue + tolerance);
```

Like in math. More readable.

Reply:
פשוט העברת את צד שמאל ימינה ואת ימין שמאלה...

Reply:
I thinks it's better to be like a ≤ x ≤ b than a ≤ x ∧ b ≥ x

Reply:
ככה נהוג לכתוב מתמטית אז בשבילו זה קריא יותר

Reply:
כאילו הכוונה שלו הייתה רק להעביר צד

Reply:
וואו עברנו לעברית פתאום